lib/assets/javascripts/dmproadmap/utils.js

```diff
@@ -20,28 +20,29 @@ $(document).ready(function(){
   });
   
   // Display tooltips when the item has focus or hover
-  $("[data-toggle='tooltip']").on('focus', function(e){
-    if($(this).attr('data-content') != undefined){
+  $("[data-toggle='tooltip']").on('click', function(e){
+    e.preventDefault();
+  });
+  $("[data-toggle='tooltip']").on('focus mouseenter', function(e){
+    e.preventDefault();
+    if($(this).attr('data-content') !== undefined){
       var y = $(this).width() + 35;
-      $(this).parent().append('<div class="tooltip-message" style="left: ' + y + 'px;">' + $(this).attr('data-content') + '</div>');
+      $(this).after('<div class="tooltip-message" style="left: ' + y + 'px;">' + $(this).attr('data-content') + '</div>');
```

Comment:
it was needed too because focus and mouseenter might be both triggered which would create the div twice.